src/components/Synthetics/PositionItem/PositionItem.tsx

```diff
@@ -303,14 +310,58 @@ export function PositionItem(p: Props) {
     return formatLiquidationPrice(p.position.liquidationPrice, { displayDecimals: indexPriceDecimals }) || "...";
   }
 
-  function renderPositionOrders() {
+  function renderPositionOrders(isSmall = false) {
     if (positionOrders.length === 0) return null;
 
     const ordersErrorList = positionOrders.filter((order) => order.errorLevel === "error");
     const ordersWarningsList = positionOrders.filter((order) => order.errorLevel === "warning");
     const hasErrors = ordersErrorList.length + ordersWarningsList.length > 0;
 
-    return (
+    function renderOrderText(order) {
+      const triggerThresholdType = getTriggerThresholdType(order.orderType, order.isLong);
+      const isIncrease = isIncreaseOrderType(order.orderType);
+      return (
+        <div key={order.key}>
+          {isDecreaseOrderType(order.orderType) ? getTriggerNameByOrderType(order.orderType, true) : t`Limit`}:{" "}
+          {triggerThresholdType}{" "}
+          {formatUsd(order.triggerPrice, {
+            displayDecimals: order.indexToken?.priceDecimals,
+          })}
+          :{" "}
+          <span>
+            {isIncrease ? "+" : "-"}
+            {formatUsd(order.sizeDeltaUsd)}
+          </span>
+        </div>
+      );
+    }
+
+    const renderMobileVersion = () => {
+      return positionOrders.map((order) => {
+        if (hasErrors) {
+          return (
+            <div className="Position-list-order">
+              <Tooltip
+                className="order-error"
+                handle={renderOrderText(order)}
+                position="right-bottom"
+                handleClassName="plain"
+                renderContent={() =>
+                  order.errors.map((error) => (
+                    <span key={error.msg} className="negative mb-xs">
+                      {error.msg}
+                    </span>
+                  ))
+                }
+              />
+            </div>
+          );
+        }
+        return <div className="Position-list-order">{renderOrderText(order)}</div>;
+      });
+    };
+
```

Comment:
I think rendering stucture is now too sophisticated:
<img width="410" alt="image" src="https://github.com/gmx-io/gmx-interface/assets/144131435/6f659ddd-cbb1-41f6-a0d4-4f53adc1635a">


All of these should be separate react components, but at least let's make the structure flat as a first step.

Reply:
also `renderOrderText` can be easily converted to react component. maybe others as well but didn't check

Reply:
I feel we should make this change on another PR. This PR contains a lot of changes and I feel it will be hard to retest all the changes.

I will create another PR

Reply:
i would agree with Anatoly that this extra nesting is undesired. not sure why we need to create separate PR for this, at first glance it should not take much time to "unnest" render functions

> This PR contains a lot of changes and I feel it will be hard to retest all the changes.

there are a lot of changes in different places, wondering why checking PositionItem component will require retesting all changes? although if it would require retesting of all changes, than the same will be required for separate PR?

Reply:
Sorry, I think it was misunderstood.

> also renderOrderText can be easily converted to react component. maybe others as well but didn't check

Based on this comment I thought to converting the render function into a components. That felt like a change to be done on a new PR so I suggested. Not just about to "unnest".

I will updated the PR.